engine/src/main/java/com/sandboni/core/engine/Application.java

```diff
@@ -40,7 +40,7 @@ public static void main(String[] args) {
         log.info("Disconnected tests to execute: {}", disconnectedTests);
         log.info("Related test suites: {}", testSuites);
 
-        log.info("Sandboni execution: {} milliseconds", TimeUtils.elapsedTime(start));
+        log.debug("Sandboni execution: {} milliseconds", TimeUtils.elapsedTime(start));
```

Comment:
Application is only used for local execution and is helpful to see how long it took to Sandboni to generate the result. I say that this should be level info.

Reply:
Noted, will keep it at level info.